### PR TITLE
Pk sidebar

### PR DESCRIPTION
**Pull request by PudparK**

### CODE REVIEW

File: src/components/HamburgerButton/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  cursor: pointer;
+  height: 34px;
+  width: 34px;
+  position: relative;
+  z-index: 101;
+  -webkit-user-select: none;
+  /* Chrome all / Safari all */
+  -moz-user-select: none;
+  /* Firefox all */
+  -ms-user-select: none;
```
Comment (josuerojasrojas) [inline src/components/HamburgerButton/styles.module.css:11]: I thought this was automatic.

File: src/components/HamburgerButton/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,145 @@
+.hamburgerButton {
+  cursor: pointer;
+  height: 34px;
+  width: 34px;
+  position: relative;
+  z-index: 101;
```
Comment (josuerojasrojas) [inline src/components/HamburgerButton/styles.module.css:6]: sort?

File: src/components/HamburgerButton/styles.module.css [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  -moz-user-select: none;
+  /* Firefox all */
+  -ms-user-select: none;
+  /* IE 10+ */
+  user-select: none;
+}
+
+.hamburgerButton span {
+  background: #ffffff;
+  width: 100%;
```
Comment (josuerojasrojas) [inline src/components/HamburgerButton/styles.module.css:18]: sort?

File: src/components/HamburgerButton/styles.module.css [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  height: 4px;
+  position: absolute;
+  left: 0;
+}
+
+.hamburgerButton span:first-child {
+  position: absolute;
+  top: 5px;
+  display: block;
```
Comment (josuerojasrojas) [inline src/components/HamburgerButton/styles.module.css:27]: sort?

File: src/components/Header/index.jsx [hunk 5]
```diff
@@ -1,18 +1,32 @@
...
+  const [toggle, setToggle] = useState(false);
 
-const Header = ({ className }) => {
   return (
-    <div className={classNames(className, styles.header)}>
-      <Link to="/">
-        <Logo imageSrc={logoSrc} />
-      </Link>
-    </div>
+    <>
```
Comment (josuerojasrojas) [inline src/components/Header/index.jsx:15]: I don't think you need fragments if you already have one div wrapping everything.

File: src/components/Header/styles.module.css [hunk 6]
```diff
@@ -1,7 +1,18 @@
...
   padding: 20px;
+  position: fixed;
   width: 100%;
+  z-index: 1000;
 }
+
+.header-space {
+  height: 76px;
+  width: 100%;
+  margin-bottom: 15px;
```
Comment (josuerojasrojas) [inline src/components/Header/styles.module.css:17]: sort?

File: src/components/Sidebar/styles.module.css [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  display: block;
+  position: relative;
+}
+
+.active::after {
+  content: "";
+  border-left: var(--green-ours) solid 4px;
+  height: 100%;
+  position: absolute;
+  left: 0;
```
Comment (josuerojasrojas) [inline src/components/Sidebar/styles.module.css:26]: sort?

File: src/components/Sidebar/styles.module.css [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.sidebar ul {
+  margin: 0;
+  padding: 0;
+}
+
+.sidebar ul li {
+  margin-bottom: 20px;
+  list-style: none;
```
Comment (josuerojasrojas) [inline src/components/Sidebar/styles.module.css:40]: sort?

File: src/components/Header/index.jsx (hunk 5 above)
Comment (PudparK) [reply src/components/Header/index.jsx:15]: Yeah but if you look closely there is another adjacent div and last I checked adjacent divs require wrapping. You can find more information on the concept at reactjs.org 🧐 .

**Pull request merged**